tests/transform/flux/test_edgar.py

```diff
@@ -257,6 +280,28 @@ def test_parse_edgar_no_domain():
                 "resolution": "0.1x0.1",
             },
         ),
+        (
```

Comment:
I would add a few options with more complicated species and source names, e.g. 
`("v7.0_FT2021_N2O_2021_TNR_Aviation_CDS.0.1x0.1.zip", {"version": "v7.0", "species": "NO2", "year": 2021, "source": "TNR-Aviation-CDS", resolution: "0.1x0.1"})` and `("v8.0_FT2022_GHG_HFC-43-10-mee_1994_TOTALS_flx.nc", {"version": "v8.0", "species": "HFC-43-10-mee", "year": 1994, "source": "TOTALS", resolution: None})`